src/main/surfrdUtilsMod.F90

```diff
@@ -145,6 +146,109 @@ subroutine convert_cft_to_pft( begg, endg, cftsize, wt_cft )
 
   end subroutine convert_cft_to_pft
 
+  !-----------------------------------------------------------------------
+  subroutine collapse_to_dom_landunits(pctwet, pctlak, pcturb, pcturb_tot, pctgla, begg, endg, n_dom_landunits, numurbl)
```

Comment:
We should also add a unit test for this subroutine

Reply:
Agreed.